includes/helper/class-wp-job-manager-helper.php

```diff
@@ -511,7 +511,7 @@ private function handle_request() {
 	 * @param string $licence_key
 	 * @param string $email
 	 */
-	private function activate_licence( $product_slug, $licence_key, $email ) {
+	public function activate_licence( $product_slug, $licence_key, $email ) {
```

Comment:
Was this method previously private for any particular reason? Just wondering if this change can open up any security breach.

Reply:
The functionality was used just inside this helper, but now we'll need it in the context of WPCOM too. I don't see any security issue here. In the end, the code is public and if someone wants to use it, they can simply copy the code and alter it the way they want.

The implementation just makes an API call to wpjobmanager.com and, if it succeeds, saves the information in an option.